aws_cloudtrail_rules/aws_codebuild_made_public.py

```diff
@@ -3,12 +3,14 @@
 
 
 def rule(event):
-    return (event['eventName'] == 'UpdateProjectVisibility' and
-        deep_get(event, 'requestParameters', 'projectVisibility') == 'PUBLIC_READ')
+    return (
+        event["eventName"] == "UpdateProjectVisibility"
```

Comment:
nit: use `.get` instead of direct key access... though every event should have an `eventName` if it's coming from CloudTrail
```suggestion
        event.get("eventName") == "UpdateProjectVisibility"
```